package.json

```
@@ -99,9 +95,7 @@
     "format:check": "prettier --check .",
     "@comment jest": "Run jest with consistent TZ and LANG",
     "jest": "cross-env NODE_ICU_DATA=node_modules/full-icu TZ=America/New_York LANG=en-GB jest --coverage",
-    "lint": "npm run lint:ts && npm run lint:css",
-    "lint:css": "stylelint **/*.{css,scss}",
-    "lint:css:fix": "stylelint **/*.{css,scss} --fix",
+    "lint": "npm run lint:ts",
     "lint:ts": "eslint . --ext .ts,.tsx",
     "lint:ts:fix": "eslint . --ext .ts,.tsx --fix",
```

Since we are not linting styles, how about:
```suggestion
    "lint": "eslint . --ext .ts,.tsx",
    "lint:fix": "eslint . --ext .ts,.tsx --fix",
```

---

I personally don't mind leaving as is.